testing/integration/agent_long_test_base_system_integ.json

```diff
@@ -0,0 +1,789 @@
+                "fsstat"
+              ],
+              "period": "1s",
+              "processors": [
+                {
+                  "drop_event.when.regexp": {
+                    "system.fsstat.mount_point": "^/(sys|cgroup|proc|dev|etc|host|lib|snap)($|/)"
+                  }
+                }
+              ]
+            }
+          },
+          {
+            "enabled": true,
+            "data_stream": {
+              "type": "metrics",
+              "dataset": "system.load"
+            },
+            "vars": {
+              "period": {
+                "value": "1s",
+                "type": "text"
+              },
+              "tags": {
+                "value": [],
+                "type": "text"
+              },
+              "processors": {
+                "type": "yaml"
+              }
+            },
+            "id": "system/metrics-system.load-9bf446fc-58d4-4767-b42d-3450815d5d3d",
+            "compiled_stream": {
+              "metricsets": [
+                "load"
+              ],
+              "condition": "${host.platform} != 'windows'",
+              "period": "1s"
+            }
+          },
+          {
+            "enabled": true,
+            "data_stream": {
+              "type": "metrics",
+              "dataset": "system.memory"
+            },
+            "vars": {
+              "period": {
+                "value": "1s",
+                "type": "text"
+              },
+              "tags": {
+                "value": [],
+                "type": "text"
+              },
+              "processors": {
+                "type": "yaml"
+              }
+            },
+            "id": "system/metrics-system.memory-9bf446fc-58d4-4767-b42d-3450815d5d3d",
+            "compiled_stream": {
+              "metricsets": [
+                "memory"
+              ],
+              "period": "1s"
+            }
+          },
+          {
+            "enabled": true,
+            "data_stream": {
+              "type": "metrics",
+              "dataset": "system.network"
+            },
+            "vars": {
+              "period": {
+                "value": "1s",
+                "type": "text"
+              },
+              "network.interfaces": {
+                "value": [],
+                "type": "text"
+              },
+              "tags": {
+                "value": [],
+                "type": "text"
+              },
+              "processors": {
+                "type": "yaml"
+              }
+            },
+            "id": "system/metrics-system.network-9bf446fc-58d4-4767-b42d-3450815d5d3d",
+            "compiled_stream": {
+              "metricsets": [
+                "network"
+              ],
+              "period": "1s",
+              "network.interfaces": null
+            }
+          },
+          {
+            "enabled": true,
+            "data_stream": {
+              "type": "metrics",
+              "dataset": "system.process"
```

Comment:
Just to double check, this is the data stream that caused the leak originally?

Reply:
Yes.